cylc/flow/etc/job.sh

```diff
@@ -132,9 +132,12 @@ cylc__job__main() {
     # System paths:
     # * workflow directory (installed run-dir first).
     export PATH="${CYLC_WORKFLOW_RUN_DIR}/bin:${PATH}"
+    export PATH="${CYLC_WORKFLOW_RUN_DIR}/share/bin:${PATH}"
     export PYTHONPATH="${CYLC_WORKFLOW_RUN_DIR}/lib/python:${PYTHONPATH:-}"
+    export PYTHONPATH="${CYLC_WORKFLOW_RUN_DIR}/share/lib/python:${PYTHONPATH:-}"
```

Comment:
Has there been any discussion on whether `/bin/` or `/share/bin/` should take precedence? With this it is currently `/share/bin/` which feels sensible enough. Worth mentioning in the docs PR?

Reply:
Good point. The more "specific" should take precedence, but which is more specific? Probably the one generated later (i.e. the share one) in which case this is correct.

Would be better to use a single `export`, note you can split lines in Bash e.g:

```bash
$ FOO="a\
> b\
> c"
[cylc] $ echo $FOO 
abc
```

Reply:
Still yet to use single imports
```suggestion
    export PATH="${CYLC_WORKFLOW_RUN_DIR}/share/bin:${CYLC_WORKFLOW_RUN_DIR}/bin:${PATH}"
    export PYTHONPATH="${CYLC_WORKFLOW_RUN_DIR}/share/lib/python:${CYLC_WORKFLOW_RUN_DIR}/lib/python:${PYTHONPATH:-}"
```

Reply:
Still yet to use single imports
```suggestion
    export PATH="${CYLC_WORKFLOW_RUN_DIR}/share/bin:${CYLC_WORKFLOW_RUN_DIR}/bin:${PATH}"
    export PYTHONPATH="${CYLC_WORKFLOW_RUN_DIR}/share/lib/python:${CYLC_WORKFLOW_RUN_DIR}/lib/python:${PYTHONPATH:-}"
```